subnet/config.go

```diff
@@ -80,7 +80,7 @@ func ParseConfig(s string) (*Config, error) {
 	}
 
 	if cfg.SubnetMax == ip.IP4(0) {
-		cfg.SubnetMax = cfg.Network.Next().IP - subnetSize
+		cfg.SubnetMax = cfg.SubnetMin + subnetSize
```

Comment:
I think the previous one is correct. And the change is incorrect. We

 should get the next network and minus the subnetSize to get the largest one in the current network.